### PR TITLE
Fix problems reported by uncrustify

### PR DESCRIPTION
**Pull request by gbiggs**
Comply with the ROS 2 style guide

### CODE REVIEW

**Comment by methylDragon** [comment]
Let's wait on the message queue PRs to resolve first, since those are massively sweeping changes? :O

File: rmw_zenoh_cpp/src/impl/client_impl.cpp [hunk 1]
```diff
@@ -11,17 +11,21 @@ std::mutex query_callback_mutex;
 std::atomic<std::int64_t> rmw_client_data_t::sequence_id_counter(0);
 std::atomic<size_t> rmw_client_data_t::client_id_counter(0);
 
+// *INDENT-OFF* because uncrustify can't decide which way to format this
```
Comment (methylDragon) [inline rmw_zenoh_cpp/src/impl/client_impl.cpp:14]: I am confused by this indentation stuff

File: rmw_zenoh_cpp/src/impl/client_impl.hpp [hunk 2]
```diff
@@ -33,13 +33,17 @@ struct rmw_client_data_t
   // Request-response sequence id (To identify and match individual requests)
   static std::atomic<std::int64_t> sequence_id_counter;
 
+  // *INDENT-OFF* because uncrustify can't decide which way to format this
```
Comment (methylDragon) [inline rmw_zenoh_cpp/src/impl/client_impl.hpp:36]: Likewise confused with the indentation stuff here

File: rmw_zenoh_cpp/src/impl/pubsub_impl.cpp [hunk 3]
```diff
@@ -16,13 +16,16 @@ std::mutex sub_callback_mutex;
 /// STATIC SUBSCRIPTION DATA MEMBERS ===========================================
 std::atomic<size_t> rmw_subscription_data_t::subscription_id_counter(0);
 
+// *INDENT-OFF* because uncrustify can't decide which way to format this
```
Comment (methylDragon) [inline rmw_zenoh_cpp/src/impl/pubsub_impl.cpp:19]: Indentation stuff :(

File: rmw_zenoh_cpp/src/impl/pubsub_impl.hpp [hunk 4]
```diff
@@ -44,9 +44,11 @@ struct rmw_subscription_data_t
   // Counter to give subscriptions unique IDs
   static std::atomic<size_t> subscription_id_counter;
 
+  // *INDENT-OFF* because uncrustify can't decide which way to format this
```
Comment (methylDragon) [inline rmw_zenoh_cpp/src/impl/pubsub_impl.hpp:47]: Indentation stuff confusion

File: rmw_zenoh_cpp/src/impl/service_impl.cpp [hunk 5]
```diff
@@ -15,13 +15,16 @@ std::mutex request_callback_mutex;
 /// STATIC SERVICE DATA MEMBERS ================================================
 std::atomic<size_t> rmw_service_data_t::service_id_counter(0);
 
+// *INDENT-OFF* because uncrustify can't decide which way to format this
```
Comment (methylDragon) [inline rmw_zenoh_cpp/src/impl/service_impl.cpp:18]: Indentation stuff confusion

File: rmw_zenoh_cpp/src/impl/service_impl.hpp [hunk 6]
```diff
@@ -28,9 +28,11 @@ struct rmw_service_data_t
   // Counter to give service servers unique IDs
   static std::atomic<size_t> service_id_counter;
 
+  // *INDENT-OFF* because uncrustify can't decide which way to format this
```
Comment (methylDragon) [inline rmw_zenoh_cpp/src/impl/service_impl.hpp:31]: Indentation stuff confusion

**Review by methylDragon — COMMENTED** [review]
Hooray for style fixes.

I'm just a bit confused about the indentation comments though.

File: rmw_zenoh_cpp/src/impl/client_impl.cpp (hunk 1 above)
Comment (gbiggs) [reply rmw_zenoh_cpp/src/impl/client_impl.cpp:14]: If you put it on one line, uncrustify tells you to wrap it.
If you wrap it without indentation, uncrustify tells you to indent it.
If you indent it, uncrustify tells you to put it on one line.

So I declared this one of those "they're guidelines, use your common sense" cases, wrapped it so it's easy to read, and told uncrustify to ignore the lines.

File: rmw_zenoh_cpp/src/impl/client_impl.hpp (hunk 2 above)
Comment (gbiggs) [reply rmw_zenoh_cpp/src/impl/client_impl.hpp:36]: Same as https://github.com/osrf/rmw_zenoh/pull/26#discussion_r486080963

File: rmw_zenoh_cpp/src/impl/pubsub_impl.cpp (hunk 3 above)
Comment (gbiggs) [reply rmw_zenoh_cpp/src/impl/pubsub_impl.cpp:19]: Same as https://github.com/osrf/rmw_zenoh/pull/26#discussion_r486080963

File: rmw_zenoh_cpp/src/impl/pubsub_impl.hpp (hunk 4 above)
Comment (gbiggs) [reply rmw_zenoh_cpp/src/impl/pubsub_impl.hpp:47]: Same as https://github.com/osrf/rmw_zenoh/pull/26#discussion_r486080963

File: rmw_zenoh_cpp/src/impl/service_impl.cpp (hunk 5 above)
Comment (gbiggs) [reply rmw_zenoh_cpp/src/impl/service_impl.cpp:18]: Same as https://github.com/osrf/rmw_zenoh/pull/26#discussion_r486080963

File: rmw_zenoh_cpp/src/impl/service_impl.hpp (hunk 6 above)
Comment (gbiggs) [reply rmw_zenoh_cpp/src/impl/service_impl.hpp:31]: Same as https://github.com/osrf/rmw_zenoh/pull/26#discussion_r486080963

File: rmw_zenoh_cpp/src/impl/client_impl.cpp (hunk 1 above)
Comment (methylDragon) [reply rmw_zenoh_cpp/src/impl/client_impl.cpp:14]: Aw, but ok

**Review by methylDragon — APPROVED** [review]
This should be good to go. I'm just wary for the indent/indent off stuff, unsure if it might cause confusion

**Comment by gbiggs** [comment]
The indent on/indent off stuff is just a [standard directive to uncrustify](https://github.com/uncrustify/uncrustify/blob/36bf34ac116a1a2e97966ebd260aefe4caebf132/src/uncrustify.cpp#L319) to tell it to ignore lines it can't handle. I don't think there will be any confusion.

**Pull request merged**